extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorLivestreamTest.java

```diff
@@ -31,6 +33,13 @@ public static void setUp() throws Exception {
         extractor.fetchPage();
     }
 
+    @Override
+    @Test
+    @Ignore("When visiting website it shows 'Lofi Girl', unknown why it's different in tests")
```

Comment:
The channel underwent a name change, maybe, and then reverted? Or it has to do with the verification process? Btw if I open the stream in NewPipe now I correctly get "Lofi Girl", it could depend on the country. Let's leave it ignored